sidechain/state/src/lib.rs

```diff
@@ -42,25 +42,6 @@ use its_primitives::{
 };
 use sp_core::H256;
 use sp_io::KillStorageResult;
-use sp_std::prelude::Vec;
-use std::marker::PhantomData;
-
-/// Sidechain wrapper and interface of the STF state.
-///
-/// TODO: In the course of refactoring the STF (#269), verify if this struct is even needed.
-/// It might be that we could implement everything directly on `[SgxExternalities]`.
```

Comment:
The changes here are exactly what the TODO comment described

Reply:
Nice, I think I remember writing that todo about 1.5 years ago! :D